### PR TITLE
feat(elixir): api to show node information, currently only version

### PR DESCRIPTION
**Pull request by hairyhum**
API worker should be initialized with the information to show This allows the managing application to chose which value to use for version

A step towards exposing current node version from Orchestrator nodes.

## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

### CODE REVIEW

File: implementations/elixir/ockam/ockam_services/test/services/node_info_api_test.exs
Comment (shanesveller) [inline implementations/elixir/ockam/ockam_services/test/services/node_info_api_test.exs:15]:
```suggestion
  test "node info without version returns error" do
```

Or "does not crash"

File: implementations/elixir/ockam/ockam_services/lib/services/api/node_info_api.ex
Comment (shanesveller) [inline implementations/elixir/ockam/ockam_services/lib/services/api/node_info_api.ex:32]: Zero functionality difference and no change necessary, but stylistically I probably would've written these as separate function heads and pattern match there. Do we have an internal style guide for that kind of super subjective stuff?

File: implementations/elixir/ockam/ockam_services/test/services/node_info_api_test.exs
Comment (hairyhum) [reply implementations/elixir/ockam/ockam_services/test/services/node_info_api_test.exs:15]: The process does crash though. `create/1` function returns error. I can validate that it crashes in the test.

File: implementations/elixir/ockam/ockam_services/lib/services/api/node_info_api.ex
Comment (hairyhum) [reply implementations/elixir/ockam/ockam_services/lib/services/api/node_info_api.ex:32]: Not sure I follow. What can you pattern match in the function head? I don't think it's possible to pattern match a proplist to get `:info` from the options.

File: implementations/elixir/ockam/ockam_services/test/services/node_info_api_test.exs
Comment (shanesveller) [reply implementations/elixir/ockam/ockam_services/test/services/node_info_api_test.exs:15]: Oh! That's an unusual design, then, but if we want to uphold that behavior as an important implementation detail, is there any way we can get a pid out somewhere in these the steps and then check for `Process.alive?` at the end?

File: implementations/elixir/ockam/ockam_services/lib/services/api/node_info_api.ex
Comment (shanesveller) [reply implementations/elixir/ockam/ockam_services/lib/services/api/node_info_api.ex:32]: 🤦🏻  I totally overlooked it was a Keyword, sorry!

**Review by shanesveller — APPROVED** [review]
LGTM, but please seek a second opinion if you have time / feel it's worthwhile. I also probably don't count for an approver yet.

**Review by mszpakowski — APPROVED** [review]

**Pull request merged**
